tools/templates/vfvs_run.py

```diff
@@ -267,19 +400,175 @@ def process_ligand(task):
+                f.writelines(['do_cavity = 0'])        
+                f.writelines(['floodfill_center = point\n'])        
+                f.writelines(['   DATA FILES'])        
+                f.writelines(['ligand_data_file {} 10'.format(task['ligand_path'])])        
+                f.writelines(['param_file = DEFAULT'])        
+                f.writelines(['set_ligand_atom_types = 1'])        
+                f.writelines(['set_protein_atom_types = 0'])        
+                f.writelines(['directory = {}'.format(conf_file.split('.')[0])])        
+                f.writelines(['tordist_file = DEFAULT'])        
+                f.writelines(['make_subdirs = 0'])        
+                f.writelines(['save_lone_pairs = 1'])        
+                f.writelines(['fit_points_file = fit_pts.mol2'])        
+                f.writelines(['read_fitpts = 0'])        
+                f.writelines(['bestranking_list_filename = bestranking.lst\n'])        
+                f.writelines(['   FLAGS'])        
+                f.writelines(['internal_ligand_h_bonds = 1'])        
+                f.writelines(['flip_free_corners = 1'])        
+                f.writelines(['match_ring_templates = 1'])        
+                f.writelines(['flip_amide_bonds = 0'])        
+                f.writelines(['flip_planar_n = 1 flip_ring_NRR flip_ring_NHR'])        
+                f.writelines(['flip_pyramidal_n = 0'])        
+                f.writelines(['rotate_carboxylic_oh = flip'])        
+                f.writelines(['use_tordist = 1'])        
+                f.writelines(['postprocess_bonds = 1'])        
+                f.writelines(['rotatable_bond_override_file = DEFAULT'])        
+                f.writelines(['solvate_all = 1\n'])        
+                f.writelines(['   TERMINATION'])        
+                f.writelines(['early_termination = 1'])        
+                f.writelines(['n_top_solutions = 3'])        
+                f.writelines(['rms_tolerance = 1.5\n'])        
+                f.writelines(['   CONSTRAINTS'])        
+                f.writelines(['force_constraints = 0\n']) 
+                f.writelines(['   COVALENT BONDING'])        
+                f.writelines(['covalent = 0\n']) 
+                f.writelines(['   SAVE OPTIONS'])        
+                f.writelines(['save_score_in_file = 1'])        
+                f.writelines(['save_protein_torsions = 1\n'])        
+                f.writelines(['  FITNESS FUNCTION SETTINGS'])        
+                f.writelines(['initial_virtual_pt_match_max = 4'])        
+                f.writelines(['relative_ligand_energy = 1'])        
+                f.writelines(['gold_fitfunc_path = goldscore'])        
+                f.writelines(['score_param_file = DEFAULT\n'])        
+                f.writelines(['  PROTEIN DATA'])    
+                f.writelines(['protein_datafile = {}'.format(conf_file['receptor'])])  
```

Comment:
conf_file is not a dict. Should this be `config_` instead of `conf_file`?

Reply:
You are correct. This is a bug that needs to be updated (will do so).